.github/workflows/label.yml

```diff
@@ -0,0 +1,17 @@
+# This workflow will triage pull requests and apply a label based on the
```

Comment:
It looks like this didn't trigger the action to run. I'm don't think GitHub allows non-org members can actually create actions on their own.  

Please let me know how to best proceed with this.